static/js/search-worker.js

```diff
@@ -6,7 +6,7 @@ var index;
 
 // Download the search index synchronously.
 var req = new XMLHttpRequest();
-req.open("GET", "/docs/search-index.json", false /* async */);
+req.open("GET", "http://localhost:8080/docs/search-index.json", false /* async */);
```

Comment:
Unintentional change.

Reply:
🙄 Thanks.

Reply:
Aside, I don't have a great way of dealing with the search index and viewing locally, but I'll typically do something like:

```
rm -rf ./public && make build && hugo serve -d ./public
```